lib/gap_intelligence/models/merchant_pricing_trend.rb

```diff
@@ -1,5 +1,11 @@
 module GapIntelligence
   class MerchantPricingTrend < Record
-    attributes :values, :headers
+    def variants
+      @variants ||= begin
+        parsed = MultiJson.load(raw[:response_body]) rescue {}
```

Comment:
@gtimti, so the main advantage we'll get will be actually in PB merchant_pricing_controller.rb.  Right now we render mpt data as json and this is actually takes half of overall time (a link to current master just in case https://github.com/GapIntelligence/pretty-bird/blob/master/app/controllers/merchant_pricing_trends_controller.rb#L19). Also, jQuery converts received response to json anyway since we specify dataType: 'json' (a link to current master just in case https://github.com/GapIntelligence/pretty-bird/blob/master/app/assets/javascripts/merchant_pricing_trend_wizard.js.coffee#L270). So if we keep only a plain response text in GapIntelligence:: MerchantPricingTrend instance and use it in PB merchant_pricing_controller, than overall response time will be significantly decreased. I completely understand that it looks not logical that we requesting data as json in MPT controller but get a text, but rendering lots of any data in rails take lot of time and is being converted to json on javascript side. 

As for calling `MultiJson.load`, parsing response with MultiJson does not take long time, but we'll end up keeping a huge amount of data in memory that we will not use those, so changes are aimed to parse response and keep only variants.

I hope that helps!